data/_models/shared/Catalogues.csv

```diff
@@ -4,4 +4,6 @@ Catalogues,,name,,1,TRUE,,,,,,http://purl.org/dc/terms/title,The name given to t
 Catalogues,,network,ref,1,TRUE,,Resources,,,,,Network or consortium that publishes this catalogue,DataCatalogueFlat,,
 Catalogues,,type,ontology,,TRUE,CatalogueOntologies,CatalogueTypes,,,,,Type of catalogue,DataCatalogueFlat,,
 Catalogues,,description,text,,TRUE,,,,,,http://purl.org/dc/terms/description,Description of this catalogue,DataCatalogueFlat,,network?.description
-Catalogues,,publisher,,,,,,,,,http://purl.org/dc/terms/publisher,The organisation responsible for making the catalogue available,DataCatalogueFlat,,
+Catalogues,,publisher,ref_array,,TRUE,,Agent,,,,http://purl.org/dc/terms/publisher,The organisation responsible for making the catalogue available,DataCatalogueFlat,,
```

Comment:
ok, this kinda make sense. I suppose that in a future refactor we should merge contacts and organisations and agent. Small proposal: we use plural everywhere, shall we rename Agent to Agents?

Reply:
Alright, will refactor all singular to plural